INSTALL.rst

```diff
@@ -11,24 +11,39 @@ Prerequisite Software
 ---------------------
 
 HaaS requires a number of packages available through the CentOS RPM
-repositories, as well as the EPEL repository. EPEL can be enabled via:
-
-::
+repositories, as well as the EPEL repository. EPEL can be enabled via::
 
     yum install epel-release
 
-Then, the rest of the packages can be installed via:
-
-::
+Then, the rest of the packages can be installed via::
 
-    yum install libvirt bridge-utils ipmitool telnet sqlite httpd mod_wsgi python-pip qemu-kvm python-virtinst
+    yum install libvirt bridge-utils ipmitool telnet httpd mod_wsgi python-pip qemu-kvm python-virtinst virt-install python-psycopg2
```

Comment:
Above you recommended installing psycopg2 via pip.

Either way is fine (install via the OS may be better since it will be updated with the system packages), but we should be consistent in which one we recommend.